service/client_state.cc

```diff
@@ -118,11 +118,11 @@ future<> service::client_state::has_keyspace_access(const sstring& ks,
 }
 
 future<> service::client_state::has_column_family_access(const sstring& ks,
-                const sstring& cf, auth::permission p) const {
+                const sstring& cf, auth::permission p, bool alter_with_opts) const {
```

Comment:
I think `auth::permission p` should be replaced with `command_desc`. p already talks about ALTER, and if we add more and more variations, the parameter list will grow.

In fact the entire parameter list can be replaced with command_desc now.

Reply:
I like that, will do.

Reply:
Done.